src/CodeGenerator/src/Definition/Shape.php

```diff
@@ -35,7 +35,7 @@ private function __construct()
 
     /**
      * @param \Closure(string, Member|null=, array<string, mixed>=): Shape $shapeLocator
-     * @param \Closure(): ServiceDefinition $serviceLocator
+     * @param \Closure(): ServiceDefinition                                $serviceLocator
```

Comment:
Why this change

Reply:
php-cs-fixer made it - it is a bugfix with the alignment when there's a complex type